react/week4/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
I believe that this test fails, as there is no `learn react` in the created component, right?

As a general rule: 
1. all PRed code needs a purpose
2. all PRs need to pass the tests

:)